addons/medical_treatment/initSettings.inc.sqf

```diff
@@ -7,6 +7,15 @@
     true
 ] call CBA_fnc_addSetting;
 
+[
+    QGVAR(numericalPulse),
+    "LIST",
+    [LSTRING(numericalPulse_DisplayName), LSTRING(NumericalPulse_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory_Treatment)],
+    [[0, 1, 2], [LSTRING(Anyone), LSTRING(Medics), LSTRING(Doctors)], 0],
```

Comment:
Default should reflect the current default imo:
```suggestion
    [[0, 1, 2], [LSTRING(Anyone), LSTRING(Medics), LSTRING(Doctors)], 1],
```

Reply:
Agreed, will do once back at computer